recipe/meta.yaml

```diff
@@ -22,9 +22,10 @@ source:
     - mingw-w64--DMS_WINXX.patch
     - win-msc_ver-1900.patch
     - fix-ABI-issue29943.patch
+    - do-not-download-externals.patch  # [win]
```

Comment:
Looking at the patch, would we be able to just set `IncludeExternals` to `false` and not need this patch or would that have other effects later on in that file?

Reply:
From what I recall, setting IncludeExtenals to false results in some modules in the standard library which depended on these external libraries not being built.  This may no longer be the case.